backend/src/test/java/org/eclipse/tractusx/puris/backend/stock/masterdata/controller/MaterialControllerTest.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2023 Volkswagen AG
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.eclipse.tractusx.puris.backend.stock.masterdata.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.eclipse.tractusx.puris.backend.common.security.SecurityConfig;
+import org.eclipse.tractusx.puris.backend.common.security.annotation.WithMockApiKey;
+import org.eclipse.tractusx.puris.backend.common.security.logic.ApiKeyAuthenticationProvider;
+import org.eclipse.tractusx.puris.backend.masterdata.controller.MaterialController;
+import org.eclipse.tractusx.puris.backend.masterdata.domain.model.Material;
+import org.eclipse.tractusx.puris.backend.masterdata.logic.dto.MaterialEntityDto;
+import org.eclipse.tractusx.puris.backend.masterdata.logic.service.MaterialService;
+import org.junit.jupiter.api.Test;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+
+
+@WebMvcTest(MaterialController.class)
+@Import({ SecurityConfig.class, ApiKeyAuthenticationProvider.class })
+public class MaterialControllerTest {
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private MaterialService materialService;
+
+    private final ModelMapper modelMapper = new ModelMapper();
+
+    @Test
+    @WithMockApiKey
+    void createMaterialTest() throws Exception {
+        // given
+        MaterialEntityDto materialDto = new MaterialEntityDto();
```

Comment:
please use more meaningful test data. You can make the materialDto / material global in this class as you reuse it multiple times